examples/python/i2c_bmp85.py

```diff
@@ -21,20 +21,21 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
+#
+# Example Usage: Shows the 'advanced' i2c functionality from python i2c
+#                read/write
 
 import mraa as m
 
-# this example will show the 'advanced' i2c functionality from python i2c
-# read/write
-
+# initialise I2C
 x = m.I2c(0)
 x.address(0x77)
 
 # initialise device
 if x.readReg(0xd0) != 0x55:
-  print("error")
+    print("error")
 
-# we want to read temperature so write 0x2e into control reg
```

Comment:
I think this comment actually makes sense, so tha tpeople understand what the example does with the sensor, without looking into the datasheet. Why do you think it needs to be removed?

Reply:
Hmm. I thought these examples are here to _just_ demonstrate the basic usage and not provide much of the device specific details. However, I agree with you. Having this line will help the user without looking into datasheet. I will keep it as it is!